### PR TITLE
Do not allow updating a resolved incident 

### PR DESCRIPTION
**Pull request by rogerclotet**

### CODE REVIEW

File: src/pages/update/Update.tsx [hunk 1]
```diff
@@ -28,6 +33,16 @@ const Update = (props: Props) => {
     return unsubscribe
   }, [props.match.params.id])
 
+  useEffect(() => {
```
Comment (Gpx) [inline src/pages/update/Update.tsx]: There's no need to run this in an effect. You can simply run it at every render, kinda like you do with a loading message.

Putting it in an effect you risk to have a short flash between the first render and the time the effect runs.

Comment (rogerclotet) [reply src/pages/update/Update.tsx]: Oh, I'll change it then :+1:

Comment (rogerclotet) [reply src/pages/update/Update.tsx]: Fixed.

File: src/pages/update/Update.tsx [hunk 2]
```diff
@@ -30,6 +35,11 @@ const Update = (props: Props) => {
 
   if (!incident) return <div>Loading...</div>
 
+  if (incident.updates.some(update => update.type === 'resolved')) {
+    // If it's already resolved, we go back to the incident status page
+    props.history.replace(`/${incident.id}`)
```
Comment (Gpx) [inline src/pages/update/Update.tsx]: FYI you could also use the `Redirect` component here. In cases where you don't have access to `history` for example

Comment (rogerclotet) [reply src/pages/update/Update.tsx]: You're right. Since I was using it inside `useEffect` I couldn't render it there, but now I can. Let me fix it.

Comment (rogerclotet) [reply src/pages/update/Update.tsx]: Done.

**Pull request merged**